src/components/TransactionFeeDetails/TransactionFeeDetails.tsx

```diff
@@ -27,6 +27,7 @@ type Props = {
   showInsufficientBalance?: boolean;
   tooltipLabel?: ReactNode;
   selectProps?: TransactionSelectTokenProps;
+  renderWrappedInTransactionDetails?: boolean;
```

Comment:
I suggest you follow the same approach as we did for bridge. We will sort a more flexible solution later on but I dunno suggest extending this component like this.